ivi/rohdeschwarz/__init__.py

```diff
@@ -0,0 +1,31 @@
+# coding=utf-8
+"""
+
+Python Interchangeable Virtual Instrument Library
+
+Copyright (c) 2018 Baiju Alexander, Acconeer Ab
```

Comment:
Capital 'b' in 'AB'?

Reply:
Also, I guess that it suffices with the company name. It will be visible in the history that you did this work anyway.